src/chains/ethereum/ethereum/tests/provider.test.ts

```diff
@@ -644,6 +646,34 @@ describe("provider", () => {
       });
     });
 
+    it("closes the logging fileDescriptor", async () => {
+      const filePath = "./closes-logging-descriptor.log";
```

Comment:
```suggestion
      const filePath = "./closes-logging-descriptor.log";
```

Should this be written to a temp location? I could imagine:

* this test failing midway through
* the file remaining
* the developer not noticing
* committing this test file to the repo
* file making its way through review

it's happened before to me. 'twas embarrassing

Reply:
Since writing the above comment we've spoken about `tmp-promise`. I agree that should be used here (and in the other tests) as well.